compiler/llvm/clangUtil.cpp

```diff
@@ -2955,6 +2955,10 @@ void makeBinaryLLVM(void) {
     command += libName;
   }
 
+  if (strcmp(CHPL_ATOMICS, "cstdlib") == 0) {
+    command += " -latomic";
+  }
```

Comment:
The only problem I see is that there doesn't seem to be a libatomic on Darwin (and we also don't see a failure there), so this needs to be conditional on the OS.

Reply:
Yeah, coincidentally I ran into that just as you made this comment. Fixed